### PR TITLE
Create .travis.yml

### PR DESCRIPTION
**Pull request by genisysram**

### CODE REVIEW

File: travis-ymls/.travis.yml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+jdk:
+  - openjdk8
+  - openjdk11
+  - openjdk15
+  - openjdk-ea
+  
+jobs:
+   exclude:
+      - arch: amd64
+        jdk: openjdk-ea
```
Comment (gururajrkatti) [inline travis-ymls/.travis.yml:43]: What is the reason this is excluded ?

**Review by gururajrkatti — COMMENTED** [review]
Name the file with the project name.

File: travis-ymls/.travis.yml (hunk 1 above)
Comment (genisysram) [reply travis-ymls/.travis.yml]: Hi updated the file name,
the Source repo has excluded  **jdk: openjdk-ea** so keeping that in allow failures

**Review by gururajrkatti — APPROVED** [review]
ok

**Pull request merged**